### PR TITLE
[ETP-385] Additional code improvements

### PR DESCRIPTION
**Pull request by cliffordp**
🎫 [ETP-385] - just some tweaks I thought of as part of https://github.com/moderntribe/event-tickets-plus/pull/1025

[ETP-385]: https://moderntribe.atlassian.net/browse/ETP-385

### CODE REVIEW

File: src/Tribe/Shortcode/Shortcode_Abstract.php [hunk 1]
```diff
@@ -38,10 +38,14 @@ abstract class Shortcode_Abstract implements Shortcode_Interface {
 	 * Example shortcode usage: [some_tag alias=17 to='Fred'] will be parsed as [some_tag canonical=17 to='Fred']
 	 *
 	 * @since 4.12.2
+	 * @since TBD Added 'event' and 'event_id' as aliases of 'post_id' by default.
 	 *
 	 * @var array<string,string>
 	 */
-	protected $aliased_arguments = [];
+	protected $aliased_arguments = [
```
Comment (sc0ttkclark) [inline src/Tribe/Shortcode/Shortcode_Abstract.php:45]: Can we get a screencast / example of this working as intended?

Comment (sc0ttkclark) [reply src/Tribe/Shortcode/Shortcode_Abstract.php:45]: Also we should check other shortcodes to make sure they don't conflict

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs artifact and confirmation there are no conflicting arguments

**Comment by borkweb** [comment]
This is stale and archived elsewhere. Closing this PR and deleting the branch.

**Pull request closed without merging**